### DISCUSSION TITLE
STM32L496VGT3 working with Micropython

### DISCUSSION THREAD

**Discussion by Ben-Anton-Utzer** [micropython/micropython#12718] [STM32 / Pyboard] [answered]
Hi guys,

I`m new in the field of Micropython and thinking about choosing a new controller. Is the STM32L496VGT3 able to run Micropython?

Cheers,
BAU

**Comment by jimmo** [comment, marked as answer]
Yes, the L496 is used in the https://www.st.com/en/evaluation-tools/32l496gdiscovery.html which MicroPython supports:
- https://micropython.org/download/STM32L496GDISC/
- https://github.com/micropython/micropython/tree/master/ports/stm32/boards/STM32L496GDISC

**Reply by water5** [reply]
STM32L496GDISCOVERY use STM32L496AG MCU (169 Pins), STM32L496VG is 100 Pins, they with same of 1 Mbytes flash memory and 320 Kbytes RAM, can I burn the STM32L496GDISC's firmware to STM32L496VG directly?
